### PR TITLE
Mimes are now Veteran Only

### PR DESCRIPTION
**Pull request by Funce**
## About The Pull Request
This makes mimes a veteran only role. 

This also adds a new module which removes all veteran only role extensions out of the main tg code.

## How This Contributes To The Skyrat Roleplay Experience
Server heath requirement. Expanding: The quality of mime roleplay has deteriorated, and it would be better served locked behind veteran crew like our Clown role.

## Changelog

:cl: Foxtrot (Funce)
balance: Mime's are now veteran only
/:cl:

### CODE REVIEW

**Comment by Stalkeros2** [comment]
Nooooo.(

**Comment by Iamgoofball** [comment]
why? "server health requirement" obviously isn't an explanation

**Comment by Lukaster42** [comment]
Did the mime slipped our host to death again?

Colonel, this is a bit too much already.

**Comment by Iamgoofball** [comment]
``The quality of mime roleplay has deteriorated``
In what ways has it deteriorated? What makes mime and it's job specifically an issue? What makes these issues not player-specific?

**Review by Gandalf2k15 — CHANGES_REQUESTED** [review]
When making copy files don't copy the directory tree structure. Just have the file in code/

**Comment by Ghommie** [comment]
> Did the mime slipped our host to death again?

I'm pretty sure they're more upset about the invisible wall than that...
Though, it's being said that the quality of mimes has dropped when, in truth, it's the host that is raising the bar on the standards of the server, notwithstanding that, if played right, mimes is a decent role to hone one's roleplaying ""skills"" through textual mimery (though some information is comprehensively too hard to convey that way, and is at the same a handicap, so I cannot blame a mime if he or she is being passive).
Basically, this looks like a judgemental grudge from the host on a role which they may deem misfit for a HRP setting, with quality being an excuse, but he may be also equating the mime with the clown, since they're both entertainer roles, with the tomfoolery of the clown projected unto the mime as a result.

A more personal opinion though: it sucks, like hell I'll ever be a "veteran" player with my history.

**Comment by alexv-ds** [comment]
As the classic said - typoe govno typogo govna. The only thing the veteran system deserves is removal.

**Comment by ChungusGamer666** [comment]
Veteran player system is a poor excuse to implement poorly balanced, poorly thought out mechanics, or at best a way to shield criticism when removing content by saying "well veterans can still do it".

**Review by Gandalf2k15 — APPROVED** [review]

**Review by GoldenAlpharex — APPROVED** [review]

**Comment by Iamgoofball** [comment]
``In what ways has it deteriorated? What makes mime and it's job specifically an issue? What makes these issues not player-specific?``

This question was never addressed.

**Comment by weesnaw-ops** [comment]
> `In what ways has it deteriorated? What makes mime and it's job specifically an issue? What makes these issues not player-specific?`
> 
> This question was never addressed.

Agreed... We've had some very, very excellent mime RPers recently who aren't veteran status. This feels dumb.

**Comment by Iamgoofball** [comment]
Asking for a followup on this.
